Find a functionally equivalent code:android.view.ViewGroup.clearDisappearingChildren:COMMENT
Method Modifier: public      
Comment:/**
 * Removes any pending animations for views that have been removed. Call
 * this if you don't want animations for exiting views to stack up.
 */

@@ -1,6 +1,15 @@
 {
-    if (mDisappearingChildren != null) {
-        mDisappearingChildren.clear();
+    final ArrayList<View> disappearingChildren = mDisappearingChildren;
+    if (disappearingChildren != null) {
+        final int count = disappearingChildren.size();
+        for (int i = 0; i < count; i++) {
+            final View view = disappearingChildren.get(i);
+            if (view.mAttachInfo != null) {
+                view.dispatchDetachedFromWindow();
+            }
+            view.clearAnimation();
+        }
+        disappearingChildren.clear();
         invalidate();
     }
 }

